Project_27/Headers/Product.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _PRODUCT_H_
+#define _PRODUCT_H_
+#include "Equipment.h"
+
+typedef struct {
+    char name[255];

```

Comment:
1 option) - read/write 255 chars
2 option - use text files and read fscanf("%s")
3 option (best) - add:
 size_t len, 
char * name;

then read len,  read next len chars....